sht3x/sht3x_example_usage.c

```diff
@@ -58,15 +53,14 @@ int main(void) {
          */
         int8_t ret = sht3x_measure_blocking_read(&temperature, &humidity);
         if (ret == STATUS_OK) {
-            /* printf("measured temperature: %0.2f degreeCelsius, "
-                      "measured humidity: %0.2f percentRH\n",
-                      temperature / 1000.0f,
-                      humidity / 1000.0f); */
+            printf("measured temperature: %0.2f degreeCelsius, "
+                   "measured humidity: %0.2f percentRH\n",
+                   temperature / 1000.0f, humidity / 1000.0f);
         } else {
-            /* printf("error reading measurement\n"); */
+            printf("error reading measurement\n");
         }
 
-        /* sleep(1); */
+        sleep(1);
```

Comment:
Can you change this to use `sensirion_sleep_usec`? So the example gets more portable.